src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -102,14 +101,18 @@
 import org.apache.olingo.commons.api.edm.FullQualifiedName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
No way this or Inject annotation can have much of an effect in this file.

Reply:
It does not, I'll go through and clean up the import statements